src/memgraph.cpp

```diff
@@ -907,10 +908,10 @@ int main(int argc, char **argv) {
     }
     db_config.durability.snapshot_interval = std::chrono::seconds(FLAGS_storage_snapshot_interval_sec);
   }
-  memgraph::storage::Storage db(db_config);
+  auto db = std::unique_ptr<memgraph::storage::Storage>(new memgraph::storage::InMemoryStorage(db_config));
```

Comment:
https://en.cppreference.com/w/cpp/memory/unique_ptr/make_unique should be a better choice here 😄